third_party/accessibility/ax/platform/ax_fragment_root_win.cc

```diff
@@ -425,4 +427,18 @@ int AXFragmentRootWin::GetIndexInParentOfChild() const {
   return 0;
 }
 
+void AXFragmentRootWin::SetAlertNode(AXPlatformNodeWin* alert_node) {
+  alert_node_ = alert_node;
```

Comment:
I do not think we have a use case for this, but setting to nullptr would remove the alert node from the root. The only time it is accessed by `AXFragmentRootWin` is in `ChildAtIndex`, where it is checked for nullity.

Reply:
If it would currently be a developer error to set it null, consider adding an `assert(alert_node != nullptr);` so we catch it in dev mode. If doing so is harmless, but unexpected, maybe just covering that in the doc comment is sufficient.